.travis.yml

```diff
@@ -16,11 +16,13 @@ before_install:
     - git clone --quiet --depth 1 https://github.com/minrk/travis-wheels travis-wheels
 install:
     - wget http://c90cb902a90d6a506ab3-a24171cbeaa42ce11788f0df3362e902.r46.cf5.rackcdn.com/pandoc-1.15.0.6.tar.gz && tar -xzf pandoc-1.15.0.6.tar.gz
-    - pip install -f travis-wheels/wheelhouse . coveralls codecov
+    - pip install -f travis-wheels/wheelhouse . codecov coverage
     - pip install nbconvert[execute,serve,test]
     - python -m ipykernel.kernelspec --user
 script:
-    - nosetests -w /tmp --with-coverage --cover-package nbconvert nbconvert
+    - # need to cd or coverage report broken.
+    - DIR=$(pwd) ; cd /tmp/
```

Comment:
I think you can use $TRAVIS_BUILD_DIR to avoid needing to store DIR again.

https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables